introduction_to_amazon_algorithms/text_classification_huggingface/Amazon_JumpStart_HF_Text_Classification.ipynb

```diff
@@ -0,0 +1,1189 @@
+{
```

Comment:
<p>SageMaker fine-tuned models -&gt; JumpStart fine-tuned models</p><p>Please remove "The inference on the dropdown models can be run in the network isolation as well. "</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3756/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
Done